ConnectionManager.lib.nut

```diff
@@ -331,8 +377,13 @@ class ConnectionManager {
         }
 
         // Run the global onConnected Handler if it exists
-        if (_onConnect != null) {
-            imp.wakeup(0, function() { _onConnect(); }.bindenv(this));
+        if (_onConnect != null && _onConnect.len()) {
+            imp.wakeup(0, function() {
```

Comment:
Questions: 
Do callback's need a priority?
Should each callback be wrapped in imp.wakeup(0) instead of the whole loop?

Reply:
done